taichi/python/export_ggui.cpp

```diff
@@ -25,7 +25,7 @@ namespace py = pybind11;
 #include "taichi/program/ndarray.h"
 #include <memory>
 
-TI_UI_NAMESPACE_BEGIN
+namespace taichi::ui {
```

Comment:
Let's do:
```
namespace taichi { 
namespace ui {
```
to preserve the coding style?

Reply:
I don't think it's necessary, and I've replaced 
```
namespace taichi {
namespace lang {
}
}
```
with 
```
namespace taichi::lang {
}
```
in the previous PR...